src/app/grievance-resolution-details/grievance-resolution-details.component.css

```
@@ -0,0 +1,21 @@
+.mat_icon_blue{
+    background-color: white;
+    min-width: 0;
+    border-radius: 50%;
+    border-color: unset !important;
+    color:#0277bd ;
+    border: unset;
```

Biome (1.9.4)

[error] Unexpected shorthand property border after border-color



(lint/suspicious/noShorthandPropertyOverrides)